cpp/src/arrow/util/logging.h

```diff
@@ -163,6 +163,9 @@ class ARROW_EXPORT ArrowLog : public ArrowLogBase {
   /// The shutdown function of arrow log, it should be used with StartArrowLog as a pair.
   static void ShutDownArrowLog();
 
+  /// Uninstall the signal actions installed by InstallFailureSignalHandler.
+  static void UninstallSignalAction();
```

Comment:
Does this really need to be a public API?

Reply:
Put to private now.